forge/src/runner.rs

```diff
@@ -43,12 +43,6 @@ pub struct TestResult {
     /// Traces
     pub traces: Vec<(TraceKind, CallTraceArena)>,
 
-    /// Debug steps
-    // TODO
-    #[serde(skip)]
-    //pub debug_calls: Option<Vec<DebugArena>>,
-    pub debug_calls: Option<Vec<()>>,
-
```

Comment:
I removed everything related to the debugger in `ContractRunner` since it is no longer used by `forge run`, and we should just help people use `forge run` to debug their tests - or alternatively wrap around `forge run` for replay capability